packages/core/entity/FileDetail/index.ts

```diff
@@ -146,6 +146,10 @@ export default class FileDetail {
         return path as string;
     }
 
+    public get isLikelyLocalFile(): boolean {
```

Comment:
This feels a little weird to me. Could we have a different condition? Are there other cases where a file doesn't start with http or s3 that the file would be cloud? will it always have that at the beginning of the path?

Reply:
Ah I see this is from before.

Reply:
We should try to add whatever Google/Microsoft's cloud platform protocol is (assuming they have one) at some point perhaps. Although I've never heard of a lab using Microsoft's so maybe just Google...

Reply:
Yeah, this was just me moving pre existing code. Happy to change to something more comprehensive

Reply:
I'll leave this as is for now. I believe the Google url protocol is `gs://`, but will hold off until we have use cases